contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -13,29 +13,32 @@ import "./DaoOperator.sol";
 
 
 /**
- * @title Kyber fee handler
+ * @title kyberFeeHandler
  *
- * @dev Kyber fee Handler works tightly with contracts KyberNetwork and KyberDao.
- *      Some events are moved to interface, easier for public uses
+ * @dev kyberFeeHandler works tightly with contracts kyberNetwork and kyberDao.
+ *      Some events are moved to interface, for easier usage
  * @dev Terminology:
- *          Epoch - KyberDao Voting campaign time frame.
- *              KyberDao voting campaigns have pre defined time period defined in number of blocks.
- *          BRR - Burn / Reward / Rebate. Kyber network fee is used for 3 purposes:
+ *          Epoch - Voting campaign time frame in kyberDao.
+ *              kyberDao voting campaigns are in the scope of epochs.
+ *          BRR - Burn / Reward / Rebate. kyberNetwork fee is used for 3 purposes:
  *              Burning KNC
- *              Reward addresse that stake KNC in KyberStaking contract. AKA - stakers
+ *              Reward an address that staked knc in kyberStaking contract. AKA - stakers
  *              Rebate reserves for supporting trades.
  * @dev Code flow:
- *      1. Accumulating && claiming Fees. Per trade on KyberNetwork, it calls handleFees() function which
+ *      1. Accumulating && claiming Fees. Per trade on kyberNetwork, it calls handleFees() function which
  *          internally accounts for network & platform fees from the trade. Fee distribution:
- *              rewards: accumulated per epoch. can be claimed by the KyberDao after epoch is concluded.
+ *              rewards: accumulated per epoch. can be claimed by the kyberDao after epoch is concluded.
  *              rebates: accumulated per rebate wallet, can be claimed any time.
- *              Burn: accumulated in the contract. Burned value and interval limited with safe check using sanity rate
+ *              Burn: accumulated in the contract. Burned value and interval limited with safe check using
+                    sanity rate.
  *              Platfrom fee: accumulated per platform wallet, can be claimed any time.
- *      2. Network Fee distribution. per epoch Kyber fee Handler reads current distribution from KyberDao.
- *          Expiry timestamp for data is set. when data expires. Fee handler reads new data from KyberDao.
+ *      2. Network Fee distribution: Per epoch kyberFeeHandler contract reads BRR distribution percentage 
+ *          from kyberDao. When the data expires, kyberFeeHandler reads updated values.
  */
 
-interface IKyberProxy is ISimpleKyberProxy, IKyberNetworkProxy { }
+interface IKyberProxy is IKyberNetworkProxy, ISimpleKyberProxy {
+    function kyberNetwork() external view returns (address);
```

Comment:
we don't need to use `kyberNetwork()` any more. This is used to get network contract from proxy contract

Reply:
right
so actually can use existing interface and not define a new one..
Good
Will remove
but not sure it belongs here.

its not clean up...

Reply:
we need to create `IKyberProxy` as combination  of IKyberNetworkProxy and ISimpleKyberProxy
we use get expected rate api from IKyberNetworkProxy
and simple swap function from ISimpleKyberProxy

Reply:
so I see now we do need the interface, since we use 1 function from each interface.
its actually kind of annoying to inherit both.

Reply:
yeah, but otherwise you have to use only `IKyberNetworkProxy` to get rate and do trade

Reply:
yes yes.